sdk/src/main/kotlin/io/rover/sdk/ui/blocks/poll/image/ImagePollOptionView.kt

```diff
@@ -88,13 +106,24 @@ internal class ImagePollOptionView(context: Context?) : LinearLayout(context) {
     }
 
     private val bottomSectionLinear = linearLayout {
+        // TODO: Change this to reflect the option style text alignment
         gravity = Gravity.CENTER
         setupRelativeLayoutParams(
             width = ViewGroup.LayoutParams.WRAP_CONTENT,
             height = bottomSectionHeight
         )
     }
 
+    private val topSectionResultsOverlayView = view {
+        id = ViewCompat.generateViewId()
```

Comment:
silly question: why do we need IDs?

Reply:
It's so that I can add `RelativeLayout` rules for certain views, although not all the views with IDs are still being used for the rules so i'll remove the views that aren't.